index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="menu.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav class="toolbar">
+        <div class="logo"><a href="">My Logo</a></div>
+        <div class="menu_main">
+            <input type="checkbox" class="toggler">
+            <div class="humbuger"><div></div></div>
+            <div class="menu">
+                <div class="overlay">
+                    <div>
+                        <ul>
+                            <li><a href="">Portifolio</a></li>
+                            <li><a href="">About</a></li>
+                            <li><a href="">Context</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+    </nav>
+    <header class="headline">
+        <div class="container">
+        <h1>I’m Patrick
+            glad to see you!</h1>
+        <p>I’m a software developer! 
+            I can help you build a product ,
+             feature or website Look through 
+             some of my work and experience! 
+             If you like what you see and have 
+             a project you need coded, 
+            don’t hestiate to contact me.</p>
+         <h3>
+            <a href="">LET’S CONNECT</a> 
+         </h3>  
+         <ul class="social-media">
+            <li><a href="twiter"><img src="../Portfolio-setup-and-mobile-first/img/twiter.svg" alt=""></a></li>
+            <li><a href="linkedin"><img src="../Portfolio-setup-and-mobile-first/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-setup-and-mobile-first/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-setup-and-mobile-first/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-setup-and-mobile-first/img/chill.svg" alt=""></a></li>
+         </ul> 
+        </div>
+         <img src="../Portfolio-setup-and-mobile-first/imgoverlay/header-shapes mobile@2x.svg" alt="">
+    </header>
+
+    
```

Comment:
- [ ] Please use an indentation of two spaces in all your documents for readability. This is a convention and also [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).